src/metabase/routes.clj

```diff
@@ -13,6 +14,14 @@
             [ring.util.response :as resp]
             [stencil.core :as stencil]))
 
+(defn- base-href []
+  (str (.getPath (clojure.java.io/as-url (public-settings/site-url))) "/"))
```

Comment:
SInce `clojure.java.io` is already required as `io` just use the alias, `(io/as-url ...)`